html/about.html

```diff
@@ -0,0 +1,191 @@
+                    </li>
+                    <li>
+                        <a href="#news">news</a>
+                    </li>
+                    <li class="campaign">
+                        <a href="#cc campaign"> cc campaign</a>
+                    </li>
+                </ul>
+            </div>
+           </div>
+        </nav>
+    </header>
+    <section class="hero-about">
+        <div class="hero-text pd-none">
+            <div class="hero-headings hero-headings-about">
+                <div class="first-part first-part-about">
+                    <p class="txt-center none">"Hello! Sharing world"</p>
+                    <h1 class="txt-center">CREATIVE COMMONS GLOBAL SUBMIT 2015</h1>
+                    <hr>
+                </div>
+                <p class="none second-part  second-part-about ">The Global Summit brings together the community of
+                experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two
+                years.
+                This year, we hope to expand our invitation list including organizations and individuals.</p>
+            </div>
+            <div class="last-txt">
+                <p class="txt-center">Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+                <div class="txt-center horizontal">
+                    ccsummit2015@cckorea.org
+                    <i class="fa-thin fa-greater-than "></i>
+                    <hr>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="creative">
+       <div class="creative-heading">
+        <h3>Creative commons global submit 2015 logo</h3>
+        <hr>
+       </div>
+        <p class="none">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</p>
+        <p class="remove">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+        After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+        After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:</p>
+        <div class="logo-div">
+            <img src="/images/about_logo 1.svg">
+        </div>
+    </section>
+    <section class="past-experiences">
+        <div class="past-experiences-container">
+            <div class="past-heading">
+                <div class="past-heading1">
+                    <h3>See Past Global Submit</h3>
+                    <hr>
+                </div>
+                <p>Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+            </div>
+            <div class="past-img">
+                <img src="/images/about_2011 1.svg">
+                <img src="/images/about_2013 1.svg">
+            </div>
```

Comment:
- [ ] Instead of showing the two images, please ensure that the text on them is actually text and not a part of the image. This eases the process of modifying it later with other text instead of having fixed images there.